stage/backup-manager/config.yaml

```diff
@@ -0,0 +1,31 @@
+metadata:
```

Comment:
I'd put this file into stage/codefresh/addons , because this product is relevant for codefresh onprem only. 
we are not going to provide common mongo backup solutions

Reply:
+ add some comments here or reference to docs/mongo-backup.md
For example, i dont see how to set mongo creds from first glance 
+ unclear not sure if longTerm creates pvc and tmp emptyDisk or maybe i'm wrong